configs/mx95libra.cfg

```
@@ -344,6 +344,7 @@ LMM_2                       ALL
 RTC                         PRIV
 SENSOR_TEMP_ANA             ALL, test
 SYS                         ALL
+BUTTON                    NOTIFY
```

Adding two spaces, so the indentation matches with the rest of the block.
```suggestion
BUTTON                      NOTIFY
```